### ISSUE TITLE
SEO enhancements | canonical tag

### ISSUE THREAD

**Issue by tu-chill** [TULiberalArts/American-Studies#9]
The canonical tag is set, but can't be reached. This may be a false positive. @lidopaglia are we setting 301s or setting canonical in .htaccess? All this might be the difference in an "s".
![screen shot 2017-04-27 at 9 56 18 am](https://cloud.githubusercontent.com/assets/12816361/25495228/6e4458b4-2b4a-11e7-89b8-5d028410f37b.png)

**Comment by tu-chill** [comment]
Google says:

Use a sitemap to indicate preferred URLs for the same content
Pick a canonical (preferred) URL for each of your pages, and tell us about your preference by submitting these canonical URLs in a sitemap.

We don't guarantee that we'll use the URLs you submit in a sitemap, but submitting one is a useful way to tell Google about the pages on your site you consider most important.

https://github.com/TULiberalArts/American-Studies/issues/10

**Comment by tu-chill** [comment]
What I just learned.

http://www.cla.temple.edu/american-studies/ -> www.cla.temple.edu/american-studies/
www.cla.temple.edu/american-studies/ -> www.cla.temple.edu/american-studies/ (obviously)
cla.temple.edu/american-studies/ -> www.cla.temple.edu/american-studies/

But:

https://www.cla.temple.edu/asian-studies/ -> stays https://www.cla.temple.edu/asian-studies/

So the links from anywhere Drupal will more than likely contain https. Drupal, like WordPress, is a security nightmare. We've also added the S to the College Menu in the TUTheme.

This lead me to look into http vs. https. 
The program pages are static pages. There is no server code or DB that is exposed. The s, for secure, is silly for sure. But then there is The Google to consider. The Google says that someday the lack of s may hurt our page rankings. It is hard to say if this is something we should be concerned about or not. 

We already have the certificates so I feel we should use https as our canonical url?
@lidopaglia thoughts?
